database/migrations/3_createEnvironmentsTable.go

```diff
@@ -16,6 +16,9 @@ func createEnvironmentsTable() *migrate.Migration {
         name varchar(255) NOT NULL,
         slug varchar(255) NOT NULL,
         application_id bigint(20) unsigned DEFAULT NULL,
+        application_uuid varchar(36) NOT NULL,
```

Comment:
We can't make changes to existing migrations now that we have deployed. Every change will need to happen in a newly created migration.